service/controller/clusterapi/v31/ebs/ebs.go

```diff
@@ -77,8 +77,11 @@ func (e *EBS) DeleteVolume(ctx context.Context, volumeID string) error {
 	return nil
 }
 
-// DetachVolume detaches an EBS volume. If force is specified data loss may occur. If shutdown is
-// specified the instance will be stopped first.
+// DetachVolume detaches an EBS volume. If force is specified data loss may
+// occur. If shutdown is specified the instance will be stopped first. Note that
+// the long term plan for the interface and functionality is to remove the EC2
+// instance management so the EBS service implementation here only manages EBS
+// volumes.
```

Comment:
Just making a note that we want to drop this support from the service implementation at some point.